app/src/main/java/edu/grinnell/appdev/grinnelldirectory/models/SimpleResult.java

```diff
@@ -0,0 +1,36 @@
+package edu.grinnell.appdev.grinnelldirectory.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import java.util.List;
+
+public class SimpleResult implements Parcelable {
+
+    List<Person> mPeople;
+
+    static Parcelable.Creator CREATOR = new Parcelable.Creator<SimpleResult>() {
```

Comment:
Could you explain what's happening here, and when this creator is used? Additionally, comments would also be helpful